lib/__tests__/shopify.js

```diff
@@ -74,9 +88,10 @@ describe('lib/shopify', () => {
 
     test('throws an error if an empty response comes back', () => {
```

Comment:
TODO: add tests for different combinations of failures from the API calls. Right now I'm _pretty sure_ everything works, but haven't tested the specifics.